akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/scaladsl/ActorTestKit.scala

```diff
@@ -215,6 +223,63 @@ final class ActorTestKit private[akka] (val name: String, val config: Config, se
    */
   def createTestProbe[M](name: String): TestProbe[M] = TestProbe(name)(system)
 
+  /**
+   * @return A test probe that is subscribed to unhandled messages from the system event bus. Subscription
+   *         will be completed and verified so any unhandled message after it will be caught by the probe.
+   */
+  def createUnhandledMessageProbe(): TestProbe[UnhandledMessage] = {
+    val probe = createTestProbe[Any]()
+    val narrowed = probe.ref.narrow[UnhandledMessage]
```

Comment:
why is this needed, instead of `createTestProbe[UnhandledMessage]()`?

Reply:
Half done thought about having some completely different message to send across bus for verification that subscription completed but I scrapped that and forgot to change.